md/single-sign-on-with-saml.md

```diff
@@ -0,0 +1,226 @@
+# Single sign-on with SAML
+
+:::info 
+**Note:** For Performance, Efficiency, and Teamwork editions only.
+:::
+
+This pages explains how to configure your Bonita BPM Platform system to use SAML to provide single sign-on (SSO). It assumes you already have a working SAML service. All Bonita BPM users must be registered in SAML.
```

Comment:
"to use the SAML protocol to provide..." would be better.
"a working SAML identity provider (IdP)"
The last sentence can be removed. As it is not true anymore